storage/bootstrap.go

```diff
@@ -194,6 +160,46 @@ func (m *bootstrapManager) Bootstrap() error {
 	return multiErr.FinalError()
 }
 
+func (m *bootstrapManager) Report() {
+	if m.IsBootstrapped() {
+		m.status.Update(1)
+	} else {
+		m.status.Update(0)
+	}
+
+}
+
+func (m *bootstrapManager) targetRanges(at time.Time) []bootstrap.TargetRange {
```

Comment:
Is it worth moving this function?  We'll lose all git blame ability to see lines that changed, etc.  

I only bring it up because its a nuanced one.